docs/router-tutorial/react-navigation-stack/react-navigation-stack.md

```diff
@@ -0,0 +1,338 @@
+---
+title: react-navigation-stack
+parent: React Navigation
+nav_order: 1
+has_children: false
+---
+
+# **React Navigation**
+
+React Navigation (RN) - related libraries that help you move between screens.
+
+## **react-navigation-stack**
+
+It's a concept that can be easily understood by anyone who's ever tried the web. Routing on the web is Stack of RN.
+
+Because RN is in the form of a mobile app, the top tab doesn't exist like a browser (chrome, Internet Explorer, etc.).
+
+Because of this, it's not easy to navigate between pages, and it also provides a 'backward button' like a browser, so remember each routing pass.
+
+And you've seen, once you've gone back in your mobile app, you've seen animation like something's piled up and gone.
+
+RN's library supporting this is 'react-navigation-stack'. However, the more deep the page is configured, the more you should be aware of the slowing that you're going to have when you're going back
+
+In this chapter, we'll try installing the react-navigation-stack, and we'll give you a quick tutorial that you can use right away.
+
+### **Environment**
+
+- Mac OS Mojave
+- [brew]([https://brew.sh/index_ko](https://brew.sh/index_ko))
+- react-native : 0.59.10 ( We don't install expo )
+- react-native-cli : 2.0.1
+- react-navigation: 4.0.10
+
+### RN install & **RN Navigation stack install**
```

Comment:
What is the meaning of RN? 
There are two famous router in react navigation. React Navigation and React Native Navigation. So, RN Navigation is not clear.

Reply:
`RN install & React Navigation Stack install`, If you say about installing `react native and react-navigation-stack`.